apps/meteor/tests/end-to-end/api/livechat/20-messages.ts

```diff
@@ -39,7 +39,7 @@ describe('LIVECHAT - messages', () => {
 
 			await sendMessage(roomId, 'Hello from visitor', token);
 			const agentMsgSentence = faker.lorem.sentence();
-			const agentMsg = await sendAgentMessage(roomId, agentMsgSentence);
+			const agentMsg = await sendAgentMessage(roomId, credentials, agentMsgSentence);
```

Comment:
Personal pref: since the function internally defaults to credentials, you can just pass undefined here.

(Actually, passing creds as the last param prevents these types of changes as most of the times you'll use the default value for this call)